### DISCUSSION TITLE
How do I force Navidrome to reload metadata from an audio file?

### DISCUSSION THREAD

**Discussion by Suoslex** [navidrome/navidrome#3264] [Q&A] [answered]
As far as I understand, Navidrome should track all changes in audio files and grab new information during scan, but by some reason it doesn't work in my case.
I use Debian 12 server, and Navidrome updates meta information only when I remove a file from library, do scan, put the file back in the library, do scan again.
On my Mac it's not a case, though. Everything works just fine.
So any advice how to force Navidrome to notice changes?

**Comment by deluan** [comment, marked as answer]
Navidrome uses timestamps to detect changes in files. Maybe the software you are using to update the tags is not updating the file's "modification time" attribute? 

You can see if the file is detected during scan by checking the logs.

**Reply by Suoslex** [reply]
Setting the modification time manually helped, thank you!
